tests/test_code_report.py

```diff
@@ -39,12 +40,24 @@ def finalize(self) -> str:
 print(f"Hello {s}.")
 """
 
+source_code_c = """
+int main() {
+\treturn 0;
+}
+"""
+
+cfg_uncrustify = """
+code_width = 120
+input_tab_size = 2
+"""
 
 log_fail = r'Found [0-9]+ issues'
 log_success = r'Issues not found'
 
 
 @pytest.mark.parametrize('analyzers, extra_args, tested_content, expected_success', [
+    [['uncrustify'], [], source_code_c, True],
```

Comment:
Please assign human-readable test id to each test instance in this or separate PR.